tests/foreman/cli/test_errata.py

```diff
@@ -204,17 +192,17 @@ def test_positive_install_by_hc_id_and_org_id(self):
         """
         install_task = HostCollection.erratum_install(
             {
-                'id': self.host_collection['id'],
-                'organization-id': self.org['id'],
-                'errata': [self.CUSTOM_ERRATA_ID],
+                'id': host_collection['id'],
+                'organization-id': module_org.id,
+                'errata': [CUSTOM_ERRATA_ID],
             }
         )
         Task.progress({'id': install_task[0]['id']})
-        for virtual_machine in self.virtual_machines:
-            self.assertTrue(self._is_errata_package_installed(virtual_machine))
+        for virtual_machine in virtual_machines:
```

Comment:
Same with `virtual_machines` - this is part of the fixture, but isn't returned by the fixture and is ambiguous in use here. I'm surprised by the fact that this doesn't result in an UnboundLocal exception on execution, as it hasn't been assigned within the function scope.

Reply:
I have reworked the fixtures, please review again